ShopAppTests/App/Cart/Views/Cells/TableView/CartTableViewCellSpec.swift

```diff
@@ -0,0 +1,200 @@
+//
+//  CartTableViewCellSpec.swift
+//  ShopAppTests
+//
+//  Created by Evgeniy Antonov on 3/9/18.
+//  Copyright © 2018 RubyGarage. All rights reserved.
+//
+
+import Nimble
+import Quick
+import ShopApp_Gateway
+
+@testable import ShopApp
+
+class CartTableViewCellSpec: QuickSpec {
+    override func spec() {
+        var cell: CartTableViewCell!
+        var quantityLabel: UILabel!
+        var quantityTextFieldView: QuantityTextFieldView!
+        var variantImageView: UIImageView!
+        var variantTitleLabel: UILabel!
+        var totalPriceLabel: UILabel!
+        var pricePerOneItemLabel: UILabel!
+        var delegateMock: CartTableCellDelegateMock!
+        
+        beforeEach {
+            let provider = CartTableProvider()
+            let tableView = UITableView()
+            tableView.dataSource = provider
+            tableView.registerNibForCell(CartTableViewCell.self)
+            
+            let indexPath = IndexPath(row: 0, section: 0)
+            let dequeuedCell: CartTableViewCell = tableView.dequeueReusableCellForIndexPath(indexPath)
+            cell = dequeuedCell
+            
+            delegateMock = CartTableCellDelegateMock()
+            cell.cellDelegate = delegateMock
+            
+            quantityLabel = self.findView(withAccessibilityLabel: "quantityLabel", in: cell) as! UILabel
+            quantityTextFieldView = self.findView(withAccessibilityLabel: "quantityTextFieldView", in: cell) as! QuantityTextFieldView
+            variantImageView = self.findView(withAccessibilityLabel: "variantImageView", in: cell) as! UIImageView
+            variantTitleLabel = self.findView(withAccessibilityLabel: "variantTitleLabel", in: cell) as! UILabel
+            totalPriceLabel = self.findView(withAccessibilityLabel: "totalPriceLabel", in: cell) as! UILabel
+            pricePerOneItemLabel = self.findView(withAccessibilityLabel: "pricePerOneItemLabel", in: cell) as! UILabel
+        }
+        
+        describe("when cell initialized") {
```

Comment:
Also test
`pricePerOneItemLabel.isHidden = true`
that setup in `didSet` of `pricePerOneItemLabel` outlet.